client/me/purchases/paths.js

```diff
@@ -6,30 +6,69 @@ export const addCreditCard = purchasesRoot + '/add-credit-card';
 
 export const billingHistory = purchasesRoot + '/billing';
 
-export function billingHistoryReceipt( receiptId = ':receiptId' ) {
+export function billingHistoryReceipt( receiptId ) {
+	if ( process.env.NODE_ENV !== 'production' ) {
+		if ( 'undefined' === typeof receiptId ) {
+			throw new Error( 'purchaseId must be provided' );
```

Comment:
The argument name is `receiptId`, so the error message could be a bit confusing.

Reply:
Fixed in c4bc73b, thanks!